src/dumper.test.ts

```diff
@@ -0,0 +1,32 @@
+import {docBuffer, ESOut, Stdout} from './dumper'
+import {generateDocs} from "./generators/endpoint";
+import {loadConfig} from "./config";
+import {Notifier} from "./notifier";
+
+beforeEach(() => {
+    while (docBuffer.length > 0) {
+        docBuffer.pop();
+    }
+});
+
+test('test StdOut dumper', async () => {
+    const dumper = new Stdout();
+    await generateDocs({endpoint_count: 3, interval: 3600, doc_count: 2});
+    await dumper.dump(docBuffer);
+    expect(true).toBeTruthy();
```

Comment:
??? 😆 

What are you testing for here? that it does not `throw`/Error?

Reply:
Yes, I did not want to change the dumper. I could have passed a console wrapper. for now not consequential.

Reply:
Feels like maybe you need to snapshot test what's in the buffer :)